_docs/2.0.0/usage/data-quick-start.md

```diff
@@ -6,7 +6,35 @@ categories:
     - '2.0.0'
     - 'usage'
 ---
+{% capture docs_path %}{{ site.baseurl }}/docs/{{ page.version }}{% endcapture %}
+
+## Prerequisites
 
 This quick start guide presumes that you have went through :
 
 - [Menas Quick Start]({{ docs_path }}/usage/menas-quick-start) guide
+
+## Data Quality
+
+Data quality is all about 2 things:
+
+- Spline
+- _INFO files
+
+## Spline
+
+but as Spline is under heavy development, we will postpone extensive documentation about it. Enceladus currently runs with version 0.3.X and it works fine out of the box. Spline 0.3.X just needs to be deployed next to Menas.
```

Comment:
**Question:** I don't think that Spline 0.3.x has to be "deployed". Perhaps only MongoDb structures needs to be created?

Reply:
Spline creates the mongo structure as it goes. And it does not have to be deployed to run as per the next paragraph.

Reply:
Actually yeah, The Spline war has to be deployed... And configured both in Menas and Enceladus
Maybe worth clarifying?

Reply:
In a quick start? I would rather not. In Deployement process `docs/2.0.0/deployment` sure thing. Can add it in this PR if you don't mind. Then maybe reference it here.

Reply:
Agreed - with the `docs/2.0.0/deployment` enhancement and referencing here.